### PR TITLE
[tabulator-tables] Various fixes across v5

### PR DESCRIPTION
**Pull request by Jojoshua**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
http://tabulator.info/docs/5.3/components#component-column
http://tabulator.info/docs/5.3/edit#editor-list
http://tabulator.info/docs/5.3/data#import
http://tabulator.info/docs/5.2/upgrade
http://tabulator.info/docs/5.3/menu

### CODE REVIEW

File: types/tabulator-tables/index.d.ts [hunk 1]
```diff
@@ -1430,6 +1428,11 @@ export interface ColumnDefinition extends ColumnLayout, CellCallbacks {
     contextMenu?: Array<MenuObject<CellComponent> | MenuSeparator> | undefined;
     clickMenu?: Array<MenuObject<CellComponent> | MenuSeparator> | undefined;
 
+    /** Popups work in a similar way to menus, but instead of only displaying lists of menu items then allow you to fill them with any custom content you like, text, input elements, forms, anything you fancy. */
```
Comment (mike-lischke) [inline types/tabulator-tables/index.d.ts:1431]: typo: "... of only displaying lists of menu items then allow you to fill them ..."

should be "... of only displaying lists of menu items they allow you to fill them ..."

But this is probably coming from the tabulator webpage, right?

**Review by mike-lischke — CHANGES_REQUESTED** [review]

File: types/tabulator-tables/index.d.ts (hunk 1 above)
Comment (Jojoshua) [reply types/tabulator-tables/index.d.ts:1431]: Yeah I typically copy from the website documentation unless it's really bad.

**Review by mike-lischke — APPROVED** [review]

**Comment by Jojoshua** [comment]
Ready to merge

File: types/tabulator-tables/index.d.ts [hunk 2]
```diff
@@ -1606,8 +1609,7 @@ export type Editor =
     | 'range'
     | 'tickCross'
     | 'star'
-    | 'select'
-    | 'autocomplete'
+    | 'list'
```
Comment (danmarshall) [inline types/tabulator-tables/index.d.ts:1612]: 'list' appears twice here - (lines 1612 & 1613) . Is 'select' deprecated?

Comment (Jojoshua) [reply types/tabulator-tables/index.d.ts:1612]: Yes select and autocomplete were replaced with list

Comment (Jojoshua) [reply types/tabulator-tables/index.d.ts:1612]: I see that list is in there twice...not sure how that happened or why TS didn't complain

**Pull request merged**